package.json

```diff
@@ -2,6 +2,9 @@
   "name": "uma.xyz",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": "~20.19.0"
```

Comment:
Is this change related?

Reply:
Yeah node v22 struggled to build nextjs v14 projects. Look me a while to figure out why my local build was failing while CI was fine.